### DISCUSSION TITLE
Is there inconsistent between document and code about SPDP phase？

### DISCUSSION THREAD

**Discussion by wddpfx** [eProsima/Fast-DDS#5153] [Q&A] [answered]
https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/simple.html#simple-initial-peers

As the document in the link say：
![1](https://github.com/user-attachments/assets/948a936c-2d62-4fe0-9545-fc886a245021)

>  the local DomainParticipant will not only send its PDP traffic to the default multicast address-port specified by its domain, but also to all the IP-port address pairs specified in the initial peers list.

But in the code. if the builtin.initialPeersList is not empty, spdp just send ParticipantProxyData to the address int the initialPeersList. If there is 3 participant A、B、C  , A`s initialPeersList just contain the address of B. And B`s initialPeersList just contain the address of A. Then How can A and C find each other? So can B and C.
![2](https://github.com/user-attachments/assets/d9b29adb-e3b6-49b9-9f18-14a106f3e48b)

**Comment by MiguelCompany** [comment, marked as answer]
This looks like a bug in the documentation.

The behavior is that announcements are sent to the list of initial peers, which defaults to the default multicast address-port specified by the domain.

If the three participants A, B, and C are in the same LAN, the default configuration will make them discover each other through multicast.
If some of the participants have a limitation with multicast, and you need to configure initial peers, you can add a UDP locator with address `239.255.0.1` and port 0.
